src/qt/coincontroldialog.cpp

```diff
@@ -533,19 +532,20 @@ void CoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
     if (model && model->getOptionsModel())
         nDisplayUnit = model->getOptionsModel()->getDisplayUnit();
 
-    QLabel *l1 = dialog->findChild<QLabel *>("labelCoinControlQuantity");
-    QLabel *l2 = dialog->findChild<QLabel *>("labelCoinControlAmount");
-    QLabel *l3 = dialog->findChild<QLabel *>("labelCoinControlFee");
-    QLabel *l4 = dialog->findChild<QLabel *>("labelCoinControlAfterFee");
-    QLabel *l5 = dialog->findChild<QLabel *>("labelCoinControlBytes");
-    QLabel *l7 = dialog->findChild<QLabel *>("labelCoinControlLowOutput");
-    QLabel *l8 = dialog->findChild<QLabel *>("labelCoinControlChange");
+
+    QLabel *l1 = ui->labelCoinControlQuantity;
+    QLabel *l2 = ui->labelCoinControlAmount;
+    QLabel *l3 = ui->labelCoinControlFee;
+    QLabel *l4 = ui->labelCoinControlAfterFee;
+    QLabel *l5 = ui->labelCoinControlBytes;
+    QLabel *l7 = ui->labelCoinControlLowOutput;
+    QLabel *l8 = ui->labelCoinControlChange;
```

Comment:
As `updateLabels()` is a member function now, we could get rid of these intermediate variables (`lN`).
This will improve code readability significantly.

Reply:
I agree but I think that refactor, and others like https://github.com/bitcoin/bitcoin/pull/17457#discussion_r348318696, could come next.

I'm also very tempted to fix other things, but I just want to ditch the static coin control to fix the issue.

Reply:
@promag Agree with your point.